### PR TITLE
feat(UserInfoHeader): Add UserInfoHeader component

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds a component for the user information to be displayed on the profile pages.

### CODE REVIEW

File: src/components/UserInfoHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  description?: string;
+  username: string;
+  link?: string;
+  sensorsCount: number;
+  recordsCount: number;
+  withEditButton?: boolean;
+  onEditButtonClick?: () => void | undefined;
+}
+
+const numberFormatter = new Intl.NumberFormat("de-DE", {
```
Comment (vogelino) [inline src/components/UserInfoHeader/index.tsx:20]: Used the Intl. Constructor to format the bigger numbers.

**Review by dnsos — APPROVED** [review]
Amazing work! Also good responsive styles.

One tiny typo should be fixed, then consider this approved!

File: src/components/UserInfoHeader/index.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          <div className='flex flex-col'>
+            <span
+              className={[
+                "text-lg sm:text-xl lg:text-2xl pt-1",
+                "font-mono lg:font-light text-gray-700",
+              ].join(" ")}
+            >
+              {numberFormatter.format(recordsCount)}
+            </span>
+            <span>Meßwerte</span>
```
[the remark this answers is not in the document]
Comment (vogelino) [reply src/components/UserInfoHeader/index.tsx:115]: Will do! Thx!

File: src/lib/urlUtil/index.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const normalizeURL = (url: string): string | undefined => {
```
[the remark this answers is not in the document]
Comment (vogelino) [reply src/lib/urlUtil/index.ts:1]: Thank you very much! 🙏

**Pull request merged**